### DISCUSSION TITLE
Should rrdcached use different rrd directory than main container ?

### DISCUSSION THREAD

**Discussion by hugalafutro** [librenms/docker#376] [Q&A]
Hi,
let me preface this by saying I've only passing understanding how most of this works. After #124 closed I re-enabled rrdcached container and noticed it created new directory `./rrd` for rrd files. Is this intended? As this restarted graphing and left over gigabytes of "old" rrd files in `./librenms/rrd`

I copied the `./librenms/rrd` files over `./rrd` and got all the old data back while keeping rrdcached working. Inspecting `examples/rrdcached-server/docker-compose.yml` I can see that it defines these for rrdcached container
```
    volumes:
      - "./rrd/db:/data/db"
      - "./rrd/journal:/data/journal"
```
but uses these for librenms container:
```
    volumes:
      - "./librenms:/data"
```
I was just wondering if these should be different or should they be same and in any of those cases why? And can I delete the `./librenms/rrd` files? Thanks!

**Comment by crazy-max** [comment]
If I recall correctly it will use internal rrdcached and write to `/data` if you don't define an external `RRDCACHED_SERVER`.
